skyvern/forge/sdk/routes/__init__.py

```
@@ -1,4 +1,5 @@
 from skyvern.forge.sdk.routes import agent_protocol  # noqa: F401
+from skyvern.forge.sdk.routes import browser_profiles  # noqa: F401
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)